src/abstractdataframe/show.jl

```diff
@@ -581,15 +581,23 @@ end
 showcols(df::AbstractDataFrame) = showcols(STDOUT, df) # -> Void
 
 using Juno
-using Juno: Inline, LazyTree, Table, Row, strong
+using Juno: Inline, LazyTree, Table, Row, strong, undefs
 
 const SIZE = 25
 
+function tomat(df::AbstractDataFrame)
```

Comment:
This function is essentially `convert(Matrix, ...)`, except that it doesn't throw when undefined entries are present. Better improve the existing method then.